examples/wrap/main.go

```diff
@@ -74,8 +75,30 @@ func main() {
 		fmt.Fprint(w, styleCss)
 	})
 	mux.HandleFunc("/wrap", func(w http.ResponseWriter, r *http.Request) {
-		// TODO(mwhittaker): Take n as a parameter.
-		wrapped, err := wrapper.Wrap(r.Context(), r.URL.Query().Get("s"), 80)
+		// Parse n.
+		n := 80
+		if nstring := r.URL.Query().Get("n"); nstring != "" {
+			var err error
+			n, err = strconv.Atoi(nstring)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusBadRequest)
+				return
+			}
+		}
+		if n <= 0 {
+			http.Error(w, fmt.Sprintf("invalid n: %d", n), http.StatusBadRequest)
+			return
+		}
+
+		// Parse s.
+		s := r.URL.Query().Get("s")
+		if len(s) > 10000 {
```

Comment:
any particular reason we put a limit on the length of the string to wrap? If so, we should be clear in specs or in the error message about this limit.

Reply:
Added the limit to the error message. I just wanted to avoid someone sending a super big string and causing the service to stall.